src/map/dungeon.hpp

```diff
@@ -30,24 +30,25 @@
 #include <vector>
 
 #include "base/savegame.hpp"
-#include "map_cell.hpp"
+#include "map/cell.hpp"
 #include "mob_creature.hpp"
 #include "util_cavegen.hpp"
 #include "util_cellular.hpp"
 #include "util_clouds.hpp"
 
 class Player;
-class LightMap;
+class map::LightMap;
```

Comment:
Rerunning the job would trigger the 1 error again:
```
/Users/runner/work/treeburner/treeburner/src/map/dungeon.hpp:40:7: error: forward declaration of class cannot have a nested name specifier
class map::LightMap;
```
Could probably be fixed like this?  I'm not sure.
```suggestion
namespace map {
class LightMap;
}  // namespace map
```
Ideally these forward references would be minimized, but that will likely be difficult.

Jobs are automatically rerun when you push another commit.

Reply:
Ah my bad. I missed this. This is generally how I have been fixing these forward references